openverse_catalog/dags/common/loader/provider_details.py

```diff
@@ -20,6 +20,7 @@
 SMITHSONIAN_DEFAULT_PROVIDER = "smithsonian"
 BROOKLYN_DEFAULT_PROVIDER = "brooklynmuseum"
 CLEVELAND_DEFAULT_PROVIDER = "clevelandmuseum"
+MET_MUSEUM_DEFAULT_PROVIDER = "met"
```

Comment:
Nice, funny this one wasn't pulled out already! Can we use the full name `METROPOLITAN_MUSEUM_DEFAULT_PROVIDER`, just for consistency?

Reply:
Done.